sources/packages/backend/libs/integrations/src/atbc-integration/models/atbc-integration.model.ts

```diff
@@ -1,4 +1,9 @@
-export interface ProcessPDRequestSummary {
-  pdRequestsProcessed: number;
-  pdRequestsUpdated: number;
+import { DisabilityStatus } from "@sims/sims-db";
+
+export interface StudentDisabilityStatusDetail {
+  sin: string;
+  lastName: string;
+  birthDate: Date;
+  disabilityStatus: DisabilityStatus;
+  disabilityStatusUpdatedDate: Date;
```

Comment:
Asking this again for clarification, the disabilityStatusUpdatedDate is an replacement of both created and updated date right?

Reply:
No. The `disabilityStatusUpdatedDate` is the property which is intended to update the `pd_date_update` eventually.
Both created and updated date still exist and we are not removing them now as a part of the ticket.